Intersect.Client/Core/Fade.cs

```diff
@@ -73,12 +90,22 @@ public static void Update()
                 {
                     sCurrentAction = FadeType.None;
                     sFadeAmt = 255f;
+
+                    if (InformServer)
```

Comment:
Move the `InformServer` check into `InformServerOfCompletion()`

Reply:
done